src/ini/config.c

```diff
@@ -0,0 +1,135 @@
+#include "config.h"
+#include "ini.h"
+
+
+int match(const char *section, const char *check_section, const char *name, const char *check_name) {
+        if (strcmp(section, check_section) == 0 && strcmp(name, check_name) == 0) {
```

Comment:
Lets import streq() and strneq() from systemd/src/fundamental/string-util-fundamental.h and create common/string.h - replacing all strcmp() except for where it's needed for hashmap. We will definitely use it more.

Reply:
I made the change can you take a look